sdk/formrecognizer/ai-form-recognizer/README.md

```diff
@@ -107,44 +103,47 @@ const client = new FormRecognizerClient("<endpoint>", new DefaultAzureCredential
+- Training custom models to recognize specific fields and values you specify by labeling your custom forms. Similarly, labeled model training returns a `CustomFormModel` indicating the fields the model will extract, as well as the model's confidence in the accuracy of each field. See the [service's documentation on labeled model training][fr-train-with-labels] for a more detailed explanation of applying labels to a training data set.
+- Managing models created in your account by creating, listing, and deleting models.
+- Copying a custom model from one Form Recognizer resource to another (or even to the same Form Recognizer resource).
+- Composing multiple custom models trained with labels into a single model. When used for custom form recognition, the new composed model will first perform a classification of the input documents to determine which of its submodels is most appropriate.
 
 Please note that models can also be trained using a graphical user interface such as the [Form Recognizer Labeling Tool][fr-labeling-tool].
 
-Sample code snippets that illustrate the use of `FormTrainingClient` can be found [below](#train-a-model).
+Sample code snippets that illustrate the use of `FormTrainingClient` can be found [below, in the "Train a Model" section.](#train-a-model).
 
 ### Long-Running Operations
-Long-running operations are operations which consist of an initial request sent to the service to start an operation,followed by polling the service at intervals to determine whether the operation has completed or failed, and if it has succeeded, to get the result.
 
-Methods that train models or extract values from forms are modeled as long-running operations.  The client exposes a `begin<operation-name>` method that returns an `Promise<PollerLike>`.  Callers should wait for the operation to complete by calling `pollUntilDone()` on the poller returned from the `begin<operation-name>` method.  Sample code snippets are provided
-to illustrate using long-running operations [below](#examples).
+Long-running operations (LROs) are operations which consist of an initial request sent to the service to start an operation, followed by polling for a result at a certain intervals to determine if the operation has completed and whether it failed or succeeded. Ultimately, the LRO will either fail with an error or produce a result.
+
+In Azure Form Recognizer, operations that train models or extract values from forms are LROs. The SDK clients provide asynchronous `begin<operation-name>` methods that return `Promise<PollerLike>` objects. The `PollerLike` object represents the LRO, and a program can wait for the operation to complete by awaiting `pollUntilDone()` on the poller returned from the `begin<operation-name>` method. Sample code snippets are provided to illustrate using long-running operations in the next section.
```

Comment:
Should we mention that copy and compose are also LROs? not sure if they fit in the "train models or extract values from forms" category.